forms/user.form.php

```diff
@@ -90,11 +89,8 @@
 		throw new InvalidArgumentException("Password cannot be empty and must be at least 8 characters");
```

Comment:
`\InvalidArguementException`

Reply:
added backslash in commit 4af650ec6a7bfcf60b55a7760a82dfe96147776e